Makefile

```diff
@@ -302,7 +302,11 @@ LDLIBS   += -lm -lgcc -lc
 include $(BOLOS_SDK)/Makefile.glyphs
 
 ### variables processed by the common makefile.rules of the SDK to grab source files and include dirs
-APP_SOURCE_PATH  += src
+APP_SOURCE_PATH  += src 
+
+APP_SOURCE_PATH  += ${BOLOS_SDK}/lib_standard_app/format.h 
```

Comment:
As discussed, I prefer
`INCLUDES_PATH  += ${BOLOS_SDK}`
Which soon will be included in the SDK makefile